sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/cosmos/CosmosProperties.java

```diff
@@ -137,4 +144,10 @@ public ConnectionMode getConnectionMode() {
     public void setConnectionMode(ConnectionMode connectionMode) {
         this.connectionMode = connectionMode;
     }
+
+    private void validateUri() {
+        if (StringUtils.startsWithIgnoreCase(uri, "mongodb://")) {
```

Comment:
How about using `allow list` instead of `blocking list`?

Reply:
which blocking list? I don't understand you.

Reply:
Your current implementation is blocking `mongodb://`.

Reply:
can you give me an example?😂

Reply:
```
private static final List<String> SUPPORTED_URI_PREFIX = Arrays.asList("foo", "bar", ...);
```